examples/server.rs

```diff
@@ -125,6 +128,14 @@ impl LanguageServer for Backend {
     fn document_highlight(&self, _: TextDocumentPositionParams) -> Self::HighlightFuture {
         Box::new(future::ok(None))
     }
+
+    fn signature_help(&self, _: TextDocumentPositionParams) -> Self::SignatureHelpFuture {
+        Box::new(future::ok(None))
+    }
+
+    fn goto_implementation(&self, _: TextDocumentPositionParams) -> Self::GotoImplementationFuture {
```

Comment:
This method should be moved to immediately below `LanguageServer::goto_type_definition()`.